src/shared/utilities/js/polyfills/array-from.js

```diff
@@ -1,5 +1,5 @@
 // Production steps of ECMA-262, Edition 6, 22.1.2.1
-if (!Array.from) {
+if (!typeof(Array.from) !== 'function') {
```

Comment:
This double negative is confusing.
See if you can change it so that the if-statement is clear what it needs to enter, instead of what it not-not wants